tangelo/algorithms/variational/tests/test_adapt_vqe_solver.py

```diff
@@ -115,6 +121,41 @@ def test_multiple_cycle_adapt_majorana_pool_with_deflation(self):
 
         self.assertAlmostEqual(optimal_energy, -1.91062, places=3)
 
+    def test_sym_projected(self):
+        """Solve Linear H3 with one frozen orbtial with ADAPTSolver using 4 cycles and operators chosen
+        from a Majorana UCCGSD pool, with a sz symmetry projection.
+        """
+
+        xyz_H3 = [
```

Comment:
Could we make this one line and stick to the same convention for floats with no decimals ?
e.g maybe `xyz_H3 = [ ('H', [0., 0., 0.]), ('H', [1., 0., 0.]), ('H', [2., 0., 0.]) ]`